boards/b-l072z-lrwan1/dist/openocd.cfg

```diff
@@ -1,3 +0,0 @@
-source [find target/stm32l0.cfg]
-
-reset_config trst_and_srst srst_nogate connect_assert_srst
```

Comment:
This reset configuration is different from the common one in `st/configs/stm32l0.cfg`

I think this file should be kept.

Reply:
> I think this file should be kept

It could but it makes no difference if using the common one (with the connect_assert_srst). I checked on other L0 based boards (nucleo-l053/nucleo-l073) and the openocd configuration file provided by this PR works for all of them.